docs/CONTRIBUTING.md

```diff
@@ -99,3 +99,13 @@ The process for contributing code is fairly simple.
 
    _This helps us ensure that we can change our internal style without breaking
    API changes._
+
+## Opening a Pull Request
+
+Atlantis uses Conventional Commits to track versions. Pull request titles should
+follow the following format.
+
+`<TYPE>(<optional SCOPE>): <conditionally BREAKING CHANGE:> <description>`
+
+Want help with your pull request title? We have a
```

Comment:
Can we embed the tool here? I want to make it as easy as possible for people to get here

Reply:
As part of this PR, can we update the PR template to link to this section as well, and remove the Conventional Commits section from the template?

Reply:
> If this is the first thing I can access when I'm making a PR, this will be really helpful, otherwise I probably won't find it
– someone who has had to rename their PRs many times

Reply:
`<conditionally BREAKING CHANGE:>` - Should we add this to the generator tool? Maybe a 

This change <select>Is not a breaking change / is a breaking change</select>

?

Reply:
1. We can't embed it in here without breaking the Github rules. Github states that a CONTRIBUTING must be a .md or .txt and it won't recognize a .mdx
2. 👍 
3: 👍 
4: a breaking change actually doesn't need to be part of the PR title. It is in the description when the PR is merged in. Any time there is a breaking change, there should be a discussion with McWeb. I don't think we need to allow for breaking changes within this tool.